### ISSUE TITLE
Track time spent in GitLab

### ISSUE THREAD

**Issue by sergey-alekseev** [toggl/toggl-button#880]
GitLab has a nice feature called [Time Tracking](https://docs.gitlab.com/ce/workflow/time_tracking.html). You can record the [time spent](https://docs.gitlab.com/ce/workflow/time_tracking.html#time-spent) working on an issue or a merge request.  

### Expected behavior
When I click the _Start timer_ button and then the _Stop timer_ button the time period should be added as _time spent_ to a corresponding issue or a merge request.

### Actual behavior
When I click the _Start timer_ button and then the _Stop timer_ button the time period isn't added as _time spent_ to a corresponding issue or a merge request.

### Steps to reproduce the behavior
0. Configure the [Gitlab Time Tracking Integration](https://toggl.com/gitlab-time-tracking)  
1. Click the _Start timer_ button  
2. Click the _Stop timer_ button  

### Additional details
![Toggl Gitlab time spent](https://user-images.githubusercontent.com/2371639/30326155-3bf502f8-97d0-11e7-9a77-123c6ffdb645.png)
![Toggl Gitlab time spent](https://user-images.githubusercontent.com/2371639/30326120-1b9c077c-97d0-11e7-9b73-e8dcd874a5c3.png)

### Environment details

* Operating system: macOS Sierra 10.12.6
* Extension version: 1.5.0

### Links
1. [Time Tracking - GitLab Documentation](https://docs.gitlab.com/ce/workflow/time_tracking.html)  
2. [Toggl - Gitlab Time Tracking Integration](https://toggl.com/gitlab-time-tracking)

**Comment by emilv** [comment]
No, this is not the expected behavior. They are two different functions. We do not want these messages clogging up our Gitlab issues:

https://user-images.githubusercontent.com/2371639/30326155-3bf502f8-97d0-11e7-9a77-123c6ffdb645.png

If you want to integrate the two systems I suggest you use the Toggl and Gitlab APIs to push tracking info to Gitlab.

**Comment by oppianmatt** [comment]
Would love for toggl to submit the time spent on the issue to the issue itself.

**Comment by egel** [comment]
@emilv I would not completely agree with you, although you're completely right about:

> They are two different functions.

Here is more about what the commenting on Gitlab really is/should be (IMHO it's really great that everything is logged - the flow is transparent).

I think @emilv, you want to filter some types of comments (they might blur a bit the view if you're working with issues/MR - agree). However, maybe developing the feature to filter the comments is better the way instead of suppressing nice new features? 😉

**Comment by tcrammond** [comment]
Thanks for all the input. I'll just add that if we were to implement a feature like this, it would probably be an opt-in setting.
